src/components/common/SimpleTable/index.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react'
 import styled from 'styled-components'
+import { MEDIA } from 'const'
+
+// TODO: move to theme AAAAND pick color for white variant
+const FONT_COLOR = '#FFFFFF';
```

Comment:
unless we expect the Table to always have these styles, we need to use theme in these places. the earlier the better else in the future we'll need a larger rework. to make it simple just add this colour scheme to all themes or sth

Reply:
Fixed.